From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Fri, 6 Oct 2023 10:46:21 -0500
Subject: [PATCH] Solus default settings

Changes:
- qt6-webengine sandboxing works fine on Solus, don't disable it
- Toast notifications don't integrate as well as native ones
---
 src/librssguard/miscellaneous/settings.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/librssguard/miscellaneous/settings.cpp b/src/librssguard/miscellaneous/settings.cpp
index b0ae74c7d..193d73ed5 100644
--- a/src/librssguard/miscellaneous/settings.cpp
+++ b/src/librssguard/miscellaneous/settings.cpp
@@ -329,7 +329,7 @@ DKEY GUI::EnableNotifications = "enable_notifications";
 DVALUE(bool) GUI::EnableNotificationsDef = true;
 
 DKEY GUI::UseToastNotifications = "use_toast_notifications";
-DVALUE(bool) GUI::UseToastNotificationsDef = true;
+DVALUE(bool) GUI::UseToastNotificationsDef = false;
 
 DKEY GUI::ToastNotificationsPosition = "toast_notifications_position";
 DVALUE(ToastNotificationsManager::NotificationPosition)
@@ -491,7 +491,7 @@ DVALUE(bool) Browser::DisableCacheDef = false;
 DKEY Browser::WebEngineChromiumFlags = "webengine_chromium_flags";
 
 #if defined(Q_OS_LINUX) && !defined(IS_FLATPAK_BUILD)
-DVALUE(QString) Browser::WebEngineChromiumFlagsDef = QSL("--no-sandbox --enable-smooth-scrolling");
+DVALUE(QString) Browser::WebEngineChromiumFlagsDef = QSL("--enable-smooth-scrolling");
 #else
 DVALUE(QString) Browser::WebEngineChromiumFlagsDef = QSL("--enable-smooth-scrolling");
 #endif
